### ISSUE TITLE
Label of train_aug_120x120

### ISSUE THREAD

**Issue by vodanhbk95** [choyingw/SynergyNet#11]
When I load file './3dmm_data/param_all_norm_v201.pkl' , I found that it has shape ([636252, 102]). I guess that 636252 is the number of image for training but I still dont understand that what 102 is?
Can you explain to me about how you can create your training set?

**Comment by choyingw** [comment]
Hi, the number of 636252 is the training data size. The 102 in order contains 12 pose, 40 shape, 10 expression, 40 textures(not used in main project, only for comparison with textures synthesized from 3DMM). So you can use the first 62 parameters to reconstruct the 3d face.

**Comment by vodanhbk95** [comment]
Big thanks. But I still wondered how you create the first 62 parameters.
Do you have the code that did it?

**Comment by choyingw** [comment]
This follows 3DDFA/ 3DDFA-V2 (https://github.com/cleardusk/3DDFA) whitening: subtract the original 3DMM parameter mean and divide it by the standard deviation to get the 62 parameters.
